setup.py

```diff
@@ -76,6 +78,17 @@ def get_config_schema():
         else:
             default_ldflags = []
 
+    if os.environ.get("CIBUILDWHEEL", False) and sys.platform == "win32":
+        import struct
+
+        default_cxxflags.append("-ID:/a/pyopencl/pyopencl/OpenCL-Headers/install/include")
+        if struct.calcsize("P") == 8:
+            default_ldflags.append("/LIBPATH:C:/Program Files/OpenCL-ICD-Loader/lib")
+            default_ldflags.append("/LIBPATH:C:/Program Files/OpenCL-ICD-Loader/bin")
+        else:
+            default_ldflags.append("/LIBPATH:C:/Program Files (x86)/OpenCL-ICD-Loader/lib")
+            default_ldflags.append("/LIBPATH:C:/Program Files (x86)/OpenCL-ICD-Loader/bin")
```

Comment:
If I should keep this if win32 builds are disabled (because of linker error)?

Reply:
Sure, doesn't hurt.